src/sonic-yang-models/yang-models/sonic-lldp.yang

```diff
@@ -0,0 +1,160 @@
+module sonic-lldp {
+    namespace "http://github.com/Azure/sonic-lldp";
+    prefix slldp;
+    yang-version 1.1;
+
+    import sonic-port {
+        prefix prt;
+    }
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONiC LLDP yang model";
+
+    revision 2020-04-16 {
+        description
+            "Initial revision.";
+    }
+
+    grouping lldp_mode_config {
+		leaf enabled {
+			type boolean;
+			description
+				"Enable/Disable LLDP";
+		}
+
+		leaf mode {
+			type enumeration {
+				enum RECEIVE;
+				enum TRANSMIT;
+			}
+			
+			description
+				"RX/TX mode for LLDP frames";
+		}
+    }
+
+    container sonic-lldp {
+        container LLDP {
+            list LLDP_LIST {
```

Comment:
single value for key and only one entry ,...should be a container,

Reply:
fixed